packages/contracts/package.json

```diff
@@ -12,22 +12,25 @@
     "coverage": "yarn hardhat coverage",
     "export-abi": "yarn run hardhat export-abi",
     "clear-abi": "yarn run hardhat clear-abi",
+
     "// progam-deploy-scripts": "//-- Program deploy Script --//",
```

Comment:
do we gain much by keeping comments as technically valid scripts? might be worthwhile removing altogether or using something like `"// progam-deploy-scripts": "echo \"this script is just a json comment\" 1>&2; exit 1",`

Reply:
Oh i like that ! Let me update that.
I just kept it to have a level of grouping (legit doesn't add in value beyond readability) 
I'll update it to your format